smb_slam/config/open3d_slam/param_rs16_localization.lua

```diff
@@ -0,0 +1,70 @@
+include "default/default_parameters.lua"
+
+--Default parameters for robosense rs16 from open3d slam
+
+params = deepcopy(DEFAULT_PARAMETERS)
+
+--ODOMETRY
+params.odometry.scan_processing.voxel_size = 0.05
+params.odometry.scan_processing.downsampling_ratio = 1.0
+params.odometry.scan_processing.scan_cropping.cropping_radius_max = 40.0
+
+
+--MAPPER_LOCALIZER
+params.mapper_localizer.is_merge_scans_into_map = false --Set this to True if you want to update the map on the fly
+params.mapper_localizer.is_build_dense_map = false
+params.mapper_localizer.is_use_map_initialization = true
+params.mapper_localizer.is_print_timing_information = true
+params.mapper_localizer.scan_to_map_registration.scan_processing.voxel_size = 0.08
+params.mapper_localizer.scan_to_map_registration.scan_processing.downsampling_ratio = 0.25
+params.mapper_localizer.scan_to_map_registration.scan_processing.scan_cropping.cropping_radius_max = 40.0
+params.mapper_localizer.scan_to_map_registration.icp.max_correspondence_dist = 0.8
+
+--MAP_INITIALIZER
+params.map_initializer.pcd_file_path = "/home/robotx/catkin_workspaces/smb_dev/src/core/smb_slam/data/maps/map.pcd"
```

Comment:
Is it necessary to have a full path here? can we just omit this param and only use a ros param for it? or have it somehow relative to the open3d_slam directory?